### PR TITLE
feat: :sparkles: powerbi refetch token

### PR DESCRIPTION
**Pull request by Gustav-Eikaas**
# Description

Please include a summary of the change and which issue is fixed.
List any external dependencies that are required for this change.

Completes:
[AB#32043](https://statoil-proview.visualstudio.com/9949ad5e-7d15-4531-901e-296574079ee1/_workitems/edit/32043)
[AB#35329](https://statoil-proview.visualstudio.com/9949ad5e-7d15-4531-901e-296574079ee1/_workitems/edit/35329)
[AB#35336](https://statoil-proview.visualstudio.com/9949ad5e-7d15-4531-901e-296574079ee1/_workitems/edit/35336)
## Checklist

-   [ ] My code follows the style guidelines of this project
-   [ ] Documentation and comments is added where needed.
-   [ ] My code is covered by tests.
-   [ ] UX has reviewed relevant UI contributions.

## Definition of Done
- [x] PR title and description are to the point and understandable
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: packages/workspace-fusion/src/lib/integrations/power-bi/utils/addPowerBi.tsx [hunk 1]
```diff
@@ -17,24 +17,29 @@ export function addPowerBi<
...
+			/>
+		),
 		CustomHeader: () => <PowerBiHeader controller={controller} />,
 		name: 'powerbi',
 		TabIcon: () => <PowerBiIcon />,
 		ignoreLoading: true,
 	});
 
-	controller.getConfig && controller.getConfig(controller.reportUri);
+	// controller.getConfig && controller.getConfig(controller.reportUri);
```
Comment (jaysencpp) [inline packages/workspace-fusion/src/lib/integrations/power-bi/utils/addPowerBi.tsx:39]: Remove commented code

File: packages/power-bi/src/lib/components/PowerBi.tsx [hunk 2]
```diff
@@ -1,72 +1,96 @@
...
+		</Suspense>
 	);
 };
 
-const PowerBiWrapper = styled.div.attrs({
-	className: 'pbiEmbed',
-})`
-	height: 100%;
-	width: 100%;
+export function Report({ getEmbedInfo, getToken, reportUri, controller, filters }: PowerBiProps) {
```
Comment (Noggling) [inline packages/power-bi/src/lib/components/PowerBi.tsx:37]: Flytt til egen fil

File: packages/power-bi/src/lib/components/loadedReport/LoadedReport.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+	const [width] = useResizeObserver(ref);
+
+	/**
+	 * TODO: Hooks for applying state while mounted
+	 */
+
+	return (
+		<StyledReportRoot ref={ref}>
+			<StyledReportContainer>
+				<div style={{ height: `${0.41 * width}px` }}>
```
Comment (Noggling) [inline packages/power-bi/src/lib/components/loadedReport/LoadedReport.tsx:23]: no inline styles

Comment (Noggling) [reply packages/power-bi/src/lib/components/loadedReport/LoadedReport.tsx:23]: make const of 0.41

**Review by JoneUE — APPROVED** [review]

**Pull request merged**